bitcoin/src/blockdata/locktime/relative.rs

```diff
@@ -209,6 +209,87 @@ impl Height {
     /// Returns the inner `u16` value.
     #[inline]
     pub fn value(self) -> u16 { self.0 }
+
+    /// Increments height by 1, saturating at `Height::MAX`.
+    #[inline]
+    pub const fn increment(self) -> Height { self.saturating_add(1) }
+
+    /// Decrements height by 1, saturating at `Height::MIN`.
+    #[inline]
+    pub const fn decrement(self) -> Height { self.saturating_sub(1) }
+
+    /// Checked integer addition.
+    ///
+    /// Computes self + rhs, returning `None` if result would overflow `Height::MAX`.
+    #[inline]
+    pub const fn checked_add(self, rhs: u16) -> Option<Self> {
+        match self.0.checked_add(rhs) {
+            Some(x) => Some(Height(x)),
+            None => None,
+        }
+    }
+
+    /// Saturating integer addition.
+    ///
+    /// Computes self + rhs, saturating at `Height::MAX` instead of overflowing.
+    #[inline]
+    pub const fn saturating_add(self, rhs: u16) -> Height { Height(self.0.saturating_add(rhs)) }
+
+    /// Checked integer subtraction.
+    ///
+    /// Computes self - rhs, returning `None` if result would overflow `Height::MIN`.
+    #[inline]
+    pub const fn checked_sub(self, rhs: u16) -> Option<Height> {
+        match self.0.checked_sub(rhs) {
+            Some(x) => Some(Height(x)),
+            None => None,
+        }
+    }
+
+    /// Saturating integer subtraction.
+    ///
+    /// Computes self - rhs, saturating at `Height::MIN` instead of overflowing.
+    #[inline]
+    pub const fn saturating_sub(self, rhs: u16) -> Height { Height(self.0.saturating_sub(rhs)) }
+
+    /// Checked integer multiplication.
+    ///
+    /// Computes self * rhs, returning `None` if result would overflow `Height::MAX`.
+    #[inline]
+    pub const fn checked_mul(self, rhs: u16) -> Option<Self> {
```

Comment:
Notably, this one is scaling and **does** actually make sense. Also other muls and divs below.